### DISCUSSION TITLE
Auth::loginUsingId() doesn't work properly, if refresh or go to  another page then Auth::id() returns NULL

### DISCUSSION THREAD

**Discussion by tfevan** [laravel/framework#36134] [Q&A]
- Laravel Version: 8.25.0
- PHP Version: 7.4.9

### Description:

If I login with `Auth::loginUsingId(1)`, then it return `1`

```
$auth = Auth::loginUsingId(1);
dd(Auth::id()); // return 1
```

But if I  go to another page then `Auth::id()` returns `NULL`. 
`Auth::loginUsingId()` doesn't work if refresh the page.

This problem only occurs with `Auth::loginUsingId()`, not occur if I login manually through `127.0.0.1:8000/login`

**Comment by crynobone** [comment]
Did you use `web` middleware on both routes?

**Reply by tfevan** [reply]
**Yes**, because the `web` middleware group is automatically applied to the default `routes.php` file by the `RouteServiceProvider`
